charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -292,19 +299,20 @@ data:
                email:
                  transportConfig:
                    transport: smtp
-                   hostname: my-smtp-server
-                   port: 587
+                   hostname: {{ printf "${%s}" .Values.rhdhOperator.secretRef.notifications_email.hostname }}
```

Comment:
why is the printf needed?

Reply:
I'm keeping the same style as done in other places that introduces env-vars into the file.
This is rendered to:
```
      - disabled: false
        package: "@redhat/plugin-notifications-backend-module-email-dynamic@0.0.0-rc.0-0"
        integrity: sha512-TikxFBxBHKJYZy8go+Mw+7yjfSJILgXjr4K0C0+tnKyMOn+OqIX6K8c1fq7IdXto3fftQ+mmCrBqJem25JjVnA==
        pluginConfig:
          notifications:
             processors:
               email:
                 transportConfig:
                   transport: smtp
                   hostname: ${NOTIFICATIONS_EMAIL_HOSTNAME}
                   port: 587
                   secure: false
                   username: ${NOTIFICATIONS_EMAIL_USERNAME}
                   password: ${NOTIFICATIONS_EMAIL_PASSWORD}
                 sender: 
                 replyTo: 
                 broadcastConfig:
                   receiver: users
                 concurrencyLimit: 10
                 cache:
                   ttl:
                     days: 1

```

Reply:
what would you suggest to produce `${NOTIFICATIONS_EMAIL_HOSTNAME}`  in this case?

Reply:
I am maybe missing something. what happens when you just use:
```
      {{ .Values.rhdhOperator.secretRef.notifications_email.hostname }}
```

Reply:
ok I was missing the { and } 
ignore then

Reply:
but why not put the ${  } in the original value?

Reply:
is this because you use the key-name someplace else?

Reply:
yes, those are keys from the backstage secret